libersoft-org/yellow-server
# fix: default user photo, disable duplicate alias/ user names

### what does this pr do?
- Disable creation of users or aliases if name is duplicate
- Display a default image if user has no photo

### how can this be manually tested?
- Try creating an alias with similar name to a user in same domain and vice-versa
- Navigate to '/webadmin/users' and view users in valid domain to see a default photo if none
